pkg/apis/pgv2.percona.com/v2/perconapgcluster_types.go

```diff
@@ -243,6 +244,10 @@ func (cr *PerconaPGCluster) Default() {
 	}
 }
 
+func (spec *PerconaPGClusterSpec) BackupsEnabled() bool {
+	return spec.Backups.Enabled == nil || *spec.Backups.Enabled
```

Comment:
I think that given that we introduce this function, we should first give this function with `Backups` receiver since it is its own responsibility to determine if it is enabled or not, and then this new function can be used here because `PerconaPGClusterSpec` knows `Backups`

Reply:
we can then use this new function here

```
func (b Backups) ToCrunchy(version string) crunchyv1beta1.Backups {
	if b.Enabled != nil && !*b.Enabled {
		return crunchyv1beta1.Backups{}
	}
```

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1021/commits/4485a3eff29345c253f110eaa4d6553f618eebe3